ansible/roles/cyhy_feeds/tasks/main.yml

```diff
@@ -25,6 +25,46 @@
       [profile elasticsearch]
       region = {{ dmarc_import_aws_region }}
 
+#
+# Create yml files for db access
+#
+
+# commander
+- name: Create cyhy.yml to store commander db credentials
+  copy:
+    dest: /var/cyhy/scripts/cyhy-feeds/cyhy.yml
+    owner: cyhy
+    group: cyhy
+    mode: 0660
+    content: |
+      database:
+        uri: mongodb://{{ commander_user }}:{{ commander_pw }}@database1.cyhy:27017/{{ commander_db }}
```

Comment:
I believe (though someone should double-check this) `cyhy-feeds` only needs read-access to the CyHy database.  If that is the case, we should use the `/cyhy/mongo/users/reporter/*` credentials, rather than the `commander` credentials (fetched [here](https://github.com/cisagov/cyhy_amis/blob/develop/ansible/roles/cyhy_feeds/vars/main.yml#L17-L22)), which have write-access.

Reply:
:eagle: :eye:, @dav3r!

Reply:
Pulled credentials and references are updated now, please let me know if I missed something.